for-loop.js

```diff
@@ -0,0 +1,57 @@
+//Challenges
+//Logging values with for loops
+//Write a for loop that logs each number from 1 - 20.
+// for (let i = 1; i < 21; i++) {
+//     console.log(i)
+// }
+//Write a for loop that logs the result of each number from 1 - 20 tripled.
+// for (let i = 1; i < 21; i++) {
+//     console.log(i * 3)
+// }
+//Create a for loop that logs each even number from 1-20, and in the place of every odd number, returns the word "ODD" Expected output --> ODD, 2, ODD, 4, ODD, 6 ...etc
+// for (let i = 1; i < 21; i += 2) {
+//     console.log(i)
+//     console.log("odd")
+// }
+
+
+// var nums = [3, 57, -9, 20, 67]
+// var largest = 0;
```

Comment:
This is a reasonable technique. Recommend to use the numbers provided in the array for the conditionals. For instance, 
var largest = nums[0];
var smallest = nums[0];
This starts the evaluation (using relational operators) by comparing each value to the 0th index of the array.